src/components/ProfilePage/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect, useContext } from "react";
+import GamesCarousel from "../GamesCarousel";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+import { Col, Layout } from "antd";
+import profilePageImg from "../../images/profilePage/profilePageImg.png";
+import db from "../../firebaseConfig";
+
+const { Sider, Content } = Layout;
+const ProfilePage = () => {
+  const { t } = useTranslation();
+  const [user, setUser] = useState("");
+  // here we hard codded the user id because the auth is not ready to work.
```

Comment:
const HARDCODED_USER = 

move this variable to line 8

add the comment above this

Reply:
PTAL